### DISCUSSION TITLE
Welcome to stator Discussions!

### DISCUSSION THREAD

**Discussion by yann510** [chocolat-chaud-io/stator#133] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KhaVNguyen** [comment]
Hey Yann! I've been learning NestJS and came across Stator. I was wondering if it's possible to only use the backend part of the template.
 I think I would prefer to have my frontend on its own repository using NextJS & Vercel's deployment/preview URL tools.

**Reply by GabrielBottari** [reply]
Hey Kha! Even though the mono repository format allows tight integration of front end and backend, the project can be only used for a or multiple backend apps. All you need to do is to delete the webapp and webapp-e2e apps in the apps folder! Cheers 🎉

**Reply by KhaVNguyen** [reply]
Hey Gabriel thanks for the reply. 
If I only opt for removing the web app (frontend) parts,do I need to configure anything else? Do we still need NX for caching since the repo won’t be as big anymore?

**Reply by yann510** [reply]
I believe if you render everything server side you wouldn't need NX. Although I think it's a good idea to keep it in case you add a new project and you want to share code across those projets.
